config/samples/kuadrant_v1beta2_authpolicy.yaml

```
@@ -0,0 +1,34 @@
+---
+apiVersion: kuadrant.io/v1beta2
+kind: AuthPolicy
+metadata:
+  name: toystore
+spec:
+  targetRef:
+    group: gateway.networking.k8s.io
+    kind: HTTPRoute
+    name: toystore
+  rules:
+    routeSelectors:
+    - matches:
+      - path:
+          type: Exact
+          value: "/admin/toy"
+        method: DELETE
```

This is an invalid example, I'm afraid.

```suggestion
  routeSelectors:
  - matches:
    - path:
        type: Exact
        value: "/admin/toy"
      method: DELETE
  rules:
```

---

Thank you. Updated the example to a simpler, similar to the RLP example.